package.json

```diff
@@ -31,9 +35,15 @@
   "dependencies": {
     "prop-types": "15.6.2"
   },
+  "peerDependencies": {
+    "react": "^16.8.6"
```

Comment:
Shouldn't we add react-native as a peer too?

Reply:
No, we don't add React-Native for the same reason we don't add react-dom.

React-Native-Web can be used instead, and npm will throw a warning every time an npm install is run.